src/test/java/com/example/demo/CheckHTTPResponse.java

```diff
@@ -0,0 +1,26 @@
+package com.example.demo;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.boot.test.web.server.LocalServerPort;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+public class CheckHTTPResponse {
+
+    @LocalServerPort
+    private int port;
+
+    @Autowired
+    private TestRestTemplate testRestTemplate;
```

Comment:
Can we use something else instead of `TestRestTemplate`?

Reply:
Do you have another test library in mind? I am not really familiar with testing libraries, but I can investigate about them

Reply:
Are we using Spring Boot 2? We can go with WebClient, instead of RestTemplate. Basically, this is not a big deal, we can start with RestTemplate :)

Reply:
I updated the branch with the GetMapping anotation